packages/design-system/src/components/UsaBanner/UsaBanner.jsx

```diff
@@ -10,14 +17,89 @@ export class UsaBanner extends React.PureComponent {
   constructor(props) {
     super(props);
     this.id = props.id || uniqueId('gov-banner_');
-    this.state = { isBannerOpen: false };
+    this.media = window.matchMedia('(max-width: 543px)');
+    this.state = {
+      isBannerOpen: false,
+      shouldRenderMobileView: this.media.matches,
+    };
     this.handleToggleBanner = this.handleToggleBanner.bind(this);
+    this.onMediaChange = this.onMediaChange.bind(this);
+
+    this.media.addEventListener('change', this.onMediaChange);
+  }
+
+  onMediaChange(evt) {
+    this.setState({ shouldRenderMobileView: evt.matches });
   }
 
   handleToggleBanner() {
     this.setState({ isBannerOpen: !this.state.isBannerOpen });
   }
 
+  // on mobile, the entire header needs to be a clickable element
+  renderMobileHeaderContent(t) {
+    return (
+      <button
+        onClick={this.handleToggleBanner}
+        className="ds-c-usa-banner__button"
+        aria-expanded={this.state.isBannerOpen}
+        aria-controls={this.id}
+      >
+        <p className="ds-c-usa-banner__header-text">
```

Comment:
Why does this icon need to be wrapped in a `<p>` with this class? Seems like it could work just as well without the wrapping element?

Reply:
Good question! I think this is left over from some copy/paste + the original markup, but while I'm refactoring, might as well clean this up too.

Reply:
looks like that class is used below as well to inline these items and set margins.. maybe `<span>` would be better.

Reply:
Looking into this and I think I can remove the `<p>` here altogether and I don't need to add the class to the SVG.